{{cookiecutter.pipeline_slug}}/Dockerfile

```diff
@@ -4,5 +4,7 @@ LABEL authors="{{ cookiecutter.author_email }}" \
     description="Docker image containing all requirements for {{ cookiecutter.pipeline_name }} pipeline"
 
 COPY environment.yml /
-RUN conda env create -f /environment.yml
-ENV PATH /opt/conda/envs/nfcore-methylseq/bin:$PATH
+RUN conda update -n base conda && \
+    conda env create -f /environment.yml && \
+    conda clean -a
+ENV PATH /opt/conda/envs/nfcore-"{{ cookiecutter.pipeline_name }}"/bin:$PATH
```

Comment:
`{{ cookiecutter.pipeline_name }}` is the full "prose" version of the pipeline name. It can contain spaces and capitalisation etc. Should instead use `pipeline_slug` and specifically avoid spaces. Also, the `nfcore-` bit is included in the pipeline name / slug.

```Dockerfile
ENV PATH /opt/conda/envs/"{{ cookiecutter.pipeline_slug.replace('/', '-') }}"/bin:$PATH
```